helm/kvm-operator/templates/configmap.yaml

```diff
@@ -56,3 +56,5 @@ data:
           ssoPublicKey: '{{ .Values.ssh.ssoPublicKey }}'
         update:
           enabled: true
+      unhealthyNodes:
+        alwaysTerminate: '{{ .Values.unhealthyNodes.alwaysTerminate }}'
```

Comment:
What about name it like AWS `terminate-unhealthy` or `terminate-unhealthy-nodes` and the use here the param `enabled` equal to `true|false` as we do with others?

Reply:
It means the flag name will change it too 🙈

Reply:
`terminateUnhealthyNodes` feels a little clearer, but I don't feel strongly if you want to keep it.

Reply:
I find it a bit misleading if it's just enabled/disabled. Because when I see it disabled I would assume that it disables the whole feature and will not terminate nodes even when the annotation is set and that's not the case. It's the other way around - if you have this enabled it will ignore the annotation and always terminate. If it's disabled it will default to using the annotation only.

Reply:
Renamed it to `terminateUnhealthyNodes`